### PR TITLE
Bug 1115626, Remove zh-TW/firefox/ redir; old rule cleanup

### PR DESCRIPTION
**Pull request by petercpg**

### CODE REVIEW

**Comment by cargoworker** [comment]
Can one of the admins verify this patch?

File: etc/httpd/global.conf [hunk 1]
```diff
@@ -32,24 +32,9 @@ RewriteRule ^(.*)$ $1 [L,R=301]
...
-RewriteCond %{REQUEST_URI} !^/zh-TW/firefox/partners
-RewriteCond %{REQUEST_URI} !^/zh-TW/firefox/desktop/tips
-RewriteCond %{REQUEST_URI} !^/zh-TW/firefox/nightly/firstrun/?
-RewriteCond %{REQUEST_URI} !^/zh-TW/firefox/hello
-RewriteCond %{REQUEST_URI} !^/zh-TW/firefox(/(?:\d+\.\d+\.?(?:\d+)?\.?(?:\d+)?(?:[a|b]?)(?:\d*)(?:pre)?(?:\d)?))?/hello/start/?
-RewriteCond %{REQUEST_URI} !^/zh-TW/firefox(/(?:\d+\.\d+\.?(?:\d+)?\.?(?:\d+)?(?:[a|b]?)(?:\d*)(?:pre)?(?:\d)?))?/firstrun/?
-RewriteCond %{REQUEST_URI} !^/zh-TW/firefox(/(?:\d+\.\d+\.?(?:\d+)?\.?(?:\d+)?(?:[a|b]?)(?:\d*)(?:pre)?(?:\d)?))?/tour/?
-RewriteCond %{REQUEST_URI} !^/zh-TW/firefox(/(?:\d+\.\d+\.?(?:\d+)?\.?(?:\d+)?(?:[a|b]?)(?:\d*)(?:pre)?(?:\d)?))?/whatsnew/?
-RewriteRule ^/zh-TW/firefox(/.*)?$ http://mozilla.com.tw/firefox$1 [L,R=301]
+RewriteCond %{REQUEST_URI} !^/zh-TW/firefox
```
Comment (pmclanahan) [inline etc/httpd/global.conf]: I don't think this line is doing anything. Remove this and then I think we're good to merge. Thanks!

File: etc/httpd/global.conf [hunk 2]
```diff
@@ -32,24 +32,9 @@ RewriteRule ^(.*)$ $1 [L,R=301]
...
 RewriteRule ^/zh-TW/mobile/?$ http://mozilla.com.tw/firefox/mobile/ [L,R=301]
 RewriteRule ^/zh-TW/download/?$ http://mozilla.com.tw/firefox/download/ [L,R=301]
 RewriteRule ^/zh-TW/products(/.*)?$ http://mozilla.com.tw/products$1 [L,R=301]
```
Comment (craigcook) [inline etc/httpd/global.conf]: The general /products/ page will soon be redirected to /firefox/products/ (see bug 1148127). No need to change the rule in this PR but I'll be changing it in another PR in the next few days.

File: etc/httpd/global.conf (hunk 1 above)
Comment (petercpg) [inline etc/httpd/global.conf]: just removed this line and squashed the commit. we are good to go. :)

**Pull request merged**